src/common/Avatar.js

```diff
@@ -22,12 +22,12 @@ const componentStyles = StyleSheet.create({
 
 type Props = {|
   avatarUrl: ?string,
-  email: string,
-  name: string,
-  size: number,
-  realm: string,
-  shape: 'square' | 'rounded' | 'circle',
-  onPress?: () => void,
+    email: string,
```

Comment:
Please use `prettier` to format code :)
Read more about setting up development environment at https://github.com/zulip/zulip-mobile/tree/master/docs

https://github.com/zulip/zulip-mobile/blob/master/docs/howto/editor.md